updates to JSFiddle examples for multiformat updates

jsnellbaker opened the pull request:
As a follow-up to the #528, these are additional updates for the multiformat changes - notably updates to the various JSFiddles used across the dev-docs/examples area of the site.

rmloveland commented:
Hi @jsnellbaker unfortunately a lot of the updated fiddles don't seem to work, and I'm not sure why.  I took screenshots of what's on master vs. what's in this PR, which I'll attach below.  It's possible I'm missing something obvious, so please let me know.

## adjust-price-577.png

![adjust-price-577](https://user-images.githubusercontent.com/1312291/35981286-e3dd8256-0cba-11e8-8df6-3b667395a334.png)

## adjust-price-master.png

![adjust-price-master](https://user-images.githubusercontent.com/1312291/35981297-e896fffc-0cba-11e8-8e44-f89f9185154f.png)

## adunit-refresh-577.png

![adunit-refresh-577](https://user-images.githubusercontent.com/1312291/35981309-f02ac870-0cba-11e8-97c3-2115f0b053b4.png)


## adunit-refresh-master.png

![adunit-refresh-master](https://user-images.githubusercontent.com/1312291/35981316-f3d4469a-0cba-11e8-9854-61fa459bd69b.png)

## basic-example-577.png

![basic-example-577](https://user-images.githubusercontent.com/1312291/35981321-f7aaf0ac-0cba-11e8-9338-4a8f27da49db.png)

## basic-example-master.png

![basic-example-master](https://user-images.githubusercontent.com/1312291/35981324-fab4921c-0cba-11e8-9230-e8a33eb229ef.png)

## brightcove-577.png

![brightcove-577](https://user-images.githubusercontent.com/1312291/35981329-fd7c8838-0cba-11e8-95af-be7e6adde288.png)

## brightcove-master.png

![brightcove-master](https://user-images.githubusercontent.com/1312291/35981333-006f7d34-0cbb-11e8-8666-c065f70d84ac.png)

## custom-price-bucket-577.png

![custom-price-bucket-577](https://user-images.githubusercontent.com/1312291/35981340-04372e08-0cbb-11e8-9da0-feb764d6fe65.png)

## custom-price-bucket-master.png

![custom-price-bucket-master](https://user-images.githubusercontent.com/1312291/35981349-098fe9d0-0cbb-11e8-934a-bb8c29a05314.png)


## custom-price-bucket-using-setpricegranularity-577.png

![custom-price-bucket-using-setpricegranularity-577](https://user-images.githubusercontent.com/1312291/35981355-0c94b20a-0cbb-11e8-9df2-271a3f74b053.png)


## custom-price-bucket-using-setpricegranularity-master.png

![custom-price-bucket-using-setpricegranularity-master](https://user-images.githubusercontent.com/1312291/35981360-10b077a2-0cbb-11e8-9078-1da85fab9778.png)


## full-page-refresh-577.png

![full-page-refresh-577](https://user-images.githubusercontent.com/1312291/35981372-1504a35a-0cbb-11e8-8626-7023fd804756.png)


## full-page-refresh-master.png

![full-page-refresh-master](https://user-images.githubusercontent.com/1312291/35981373-17e89c20-0cbb-11e8-9dbd-04b9ddb04bc9.png)

jsnellbaker commented:
Hi @rmloveland, I went through the various fiddles and found the issue.  These pages were pointing to the legacy 0.34.x branch instead of the current 1.x branch, which contains the functionality to use this new syntax style.  I went through all the fiddles again and converted them to the 1.x format and verified they are working again.

Note as part of these updates, certain pages had other aspects of the code also updated to the 1.x version of the functionality (such as using `setConfig()` instead of using a new depreciated method directly).

Please let me know if you have questions or still see any problems.  Thanks.
